web-view/viewer/src/jsMain/kotlin/com/firsttimeinforever/intellij/pdf/viewer/application/Application.kt

```diff
@@ -220,27 +220,6 @@ class Application(private val viewer: ViewerAdapter) {
   }
 
   private fun rebindStandardEvents() {
-    viewer.viewerApp.asDynamic().unbindWindowEvents()
-    val boundEvents = viewer.viewerApp.asDynamic()._boundEvents
-    boundEvents.windowResize = {
-      viewer.viewerApp.eventBus.dispatch("resize", json("source" to window))
-    }
-    boundEvents.windowHashChange = {
-      viewer.viewerApp.eventBus.dispatch("hashchange", json(
-        "source" to window,
-        "hash" to document.location?.hash?.substring(1)
-      ))
-    }
-    boundEvents.windowBeforePrint = {
-      viewer.viewerApp.eventBus.dispatch("beforeprint", json("source" to window))
-    }
-    boundEvents.windowAfterPrint = {
-      viewer.viewerApp.eventBus.dispatch("afterprint", json("source" to window))
-    }
-    window.addEventListener("resize", boundEvents.windowResize.unsafeCast<EventListener>())
-    window.addEventListener("hashchange", boundEvents.windowHashChange.unsafeCast<EventListener>())
-    window.addEventListener("beforeprint", boundEvents.windowBeforePrint.unsafeCast<EventListener>())
-    window.addEventListener("afterprint", boundEvents.windowAfterPrint.unsafeCast<EventListener>())
```

Comment:
we should think about this, use a better way to sync state.

Reply:
I agree this is not ideal, but what other way would you have in mind? We need to handle keyboard events in some way and link them to the viewer state.

Reply:
The previous implementation for them is dispatch on event bus, and the default handler is event bus too.

When I scroll between page, the ViewState is changed, however, no subscriber of `PdfViewStateChangedListener.TOPIC` works (it's in PdfDocumentPageStatusBarWidget, however, no log at all).

Reply:
Ah, the id of PdfDocumentPageStatusBarWidget was incorrect so the widget was broken, fixed that in #115. Well spotted, thanks!